main/activities.js

```diff
@@ -26,14 +26,13 @@ class Activities {
   #detectChangeInOnlineStatus (activity) {
     if (
       activity.type === 'info' &&
-      activity.message.includes('Saturn Node is online')
+      (activity.message === 'SPARK started reporting retrievals' ||
+      activity.message === 'SPARK retrieval reporting resumed')
     ) {
+      console.log('XXX went online')
```

Comment:
Why `XXX` and not `SPARK`?

Reply:
ooops that was a debug call. will remove

Reply:
c274852